pyproject.toml

```diff
@@ -42,10 +39,10 @@
 
 
         flake8-django            = "^1.1"
-        flake8-logging-format    = "^0.6"
```

Comment:
the only thing this plugin gives is enforce `extra` param instead of python string formats and I dont think it's necessary